dataline-integration-tests/src/test/java/io/dataline/integration_tests/destinations/TestBigQueryDestination.java

```diff
@@ -0,0 +1,184 @@
+
+  private static final BigQuery BQ = BigQueryOptions.getDefaultInstance().getService();
+
+  private String datasetName;
+  private Dataset dataset;
+  private Process process;
+
+  @BeforeEach
+  public void setUpBigQuery() throws IOException {
+    datasetName = "dataline_tests_" + RandomStringUtils.randomAlphanumeric(8);
+    DatasetInfo datasetInfo = DatasetInfo.newBuilder(datasetName).build();
+    dataset = BQ.create(datasetInfo);
+    System.out.println("BQ Dataset " + datasetName + " created...");
+
+    writeConfigFileToJobRoot();
+    process = startTarget();
+  }
+
+  @AfterEach
+  public void tearDownBigQuery() {
+    WorkerUtils.closeProcess(process);
+
+    // allows deletion of a dataset that has contents
+    BigQuery.DatasetDeleteOption option = BigQuery.DatasetDeleteOption.deleteContents();
+
+    boolean success = BQ.delete(dataset.getDatasetId(), option);
+    if (success) {
+      System.out.println("BQ Dataset " + datasetName + " deleted...");
+    } else {
+      System.out.println("BQ Dataset cleanup for " + datasetName + " failed!");
+    }
+  }
+
+  @Test
+  public void runTest() throws IOException, InterruptedException {
+    writeResourceToStdIn("singer-tap-output.txt", process);
+
+    await()
+        .atMost(10, SECONDS)
+        .until(
+            () -> {
+              try {
+                return getExchangeRateTable().getTotalRows() == 2L;
+              } catch (Exception e) {
+                return false;
+              }
+            });
+
+    TableResult results = getExchangeRateTable();
+    List<FieldValueList> actualList =
+        StreamSupport.stream(results.iterateAll().spliterator(), false).collect(toList());
+
+    List<FieldValueList> expectedList =
+        Arrays.asList(
+            FieldValueList.of(
+                Arrays.asList(fieldValue(1598918400.0), fieldValue("0.0"), fieldValue("2.0"))),
+            FieldValueList.of(
+                Arrays.asList(fieldValue(1598918400.0), fieldValue("1.0"), fieldValue("7.0"))));
+
+    for (int i = 0; i < expectedList.size(); i++) {
+      FieldValueList actual = actualList.get(i);
+      FieldValueList expected = expectedList.get(i);
+
+      System.out.println("actual = " + actual);
+      System.out.println("expected = " + expected);
+
+      // compare all except datetime
+      for (int j = 1; j < 3; j++) {
+        assertEquals(expected.get(j).getValue(), actual.get(j).getValue());
+      }
+    }
+  }
+
+  private Process startTarget() throws IOException {
+    return pbf.create(
+        jobRoot,
+        "dataline/integration-singer-bigquery-destination",
+        "--config",
+        "rendered_bigquery.json")
+        .redirectOutput(ProcessBuilder.Redirect.INHERIT)
+        .redirectError(ProcessBuilder.Redirect.INHERIT)
+        .start();
+  }
+
+  private void writeConfigFileToJobRoot() throws IOException {
+
+    String partialConfigString = new String(Files.readAllBytes(Paths.get("config/bigquery.json")));
```

Comment:
is there supposed to be anything in this path already? I'm not seeing it populated elsewhere